stripper/ze_last_man_standing_p3.cfg

```diff
@@ -732,3 +1126,442 @@ modify:
+	"OnTrigger" "ex5_music_textAddOutputmessage Dark nights on my soul1361"
+	"OnTrigger" "ex5_music_textDisplay136.11"
+	"OnTrigger" "ex5_music_textAddOutputcolor 255 55 01601"
+	"OnTrigger" "ex5_music_textAddOutputmessage I deny failure1611"
+	"OnTrigger" "ex5_music_textDisplay161.51"
+	"OnTrigger" "ex5_music_textAddOutputmessage I ignite1631"
+	"OnTrigger" "ex5_music_textDisplay163.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Woe is on my misery1651"
+	"OnTrigger" "ex5_music_textDisplay165.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage She wins all their eyes1681"
+	"OnTrigger" "ex5_music_textDisplay168.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Realise what defies our fate1701"
+	"OnTrigger" "ex5_music_textDisplay170.51"
+	"OnTrigger" "ex5_music_textAddOutputmessage This is not me this is me1731"
+	"OnTrigger" "ex5_music_textDisplay173.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage So if I struggle a lifetime1761"
+	"OnTrigger" "ex5_music_textDisplay176.51"
+	"OnTrigger" "ex5_music_textAddOutputmessage What good would that do?1791"
+	"OnTrigger" "ex5_music_textDisplay179.11"
+	"OnTrigger" "ex5_music_textAddOutputcolor 255 0 01821"
+	"OnTrigger" "ex5_music_textAddOutputmessage If I got a plan1821"
+	"OnTrigger" "ex5_music_textDisplay182.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Doesn't have to stop the feeling inside1881"
+	"OnTrigger" "ex5_music_textDisplay188.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage If I do make sense1931"
+	"OnTrigger" "ex5_music_textDisplay193.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Would you drag me down?1991"
+	"OnTrigger" "ex5_music_textDisplay199.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Cause nothing lasts forever...!2031"
+	"OnTrigger" "ex5_music_textDisplay203.51"
+	"OnTrigger" "ex5_music_textAddOutputcolor 255 0 002311"
+	"OnTrigger" "ex5_music_textAddOutputmessage How do I get home?2311"
+	"OnTrigger" "ex5_music_textDisplay231.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Everything revolves around me2341"
+	"OnTrigger" "ex5_music_textDisplay234.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage If I can't find myself?2361"
+	"OnTrigger" "ex5_music_textDisplay236.51"
+	"OnTrigger" "ex5_music_textAddOutputmessage If so Its so completely fake2401"
+	"OnTrigger" "ex5_music_textDisplay240.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage How do I get home?2421"
+	"OnTrigger" "ex5_music_textDisplay242.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Everything revolves around me2451"
+	"OnTrigger" "ex5_music_textDisplay245.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage If even you can't help?2471"
+	"OnTrigger" "ex5_music_textDisplay247.11"
+	"OnTrigger" "ex5_music_textAddOutputmessage Dark nights on my soul2511"
+	"OnTrigger" "ex5_music_textDisplay251.51"
+}
```

Comment:
Not interested in taking this, as it doesn't make sense to force more game_text server-side with how broken the entity is on CS:GO.

Reply:
Ok.